docs/reporters/AntennaAttributionDocumentReporter.md

```diff
@@ -0,0 +1,50 @@
+# AntennaAttributionDocumentReporter
+
+The AntennaAttributionDocumentReporter generates an attribution document in PDF format by leveraging the [Eclipse 
+Antenna][1] project. The attribution document can be customized by providing a template bundle JAR, or a directory with 
+template PDF files. 
+
+There is a guide in the [Antenna project documentation][2] which describes how such a template bundle JAR file for the 
+customization has to be created before using the reporter. 
```

Comment:
How about saying just: "..which describes how to create such a template bundle JAR file."

"before using the reporter" is a repetition imo as the previous sentence already states that the template is an input to the reporter.

Consider making this sentence part of above block, e.g. remove the empty / new line above.

Reply:
Done.